osm2lanes/src/lib.rs

```diff
@@ -571,7 +583,8 @@ impl MapConfig {
             country_code: String::new(),
             bikes_can_use_bus_lanes: true,
             inferred_sidewalks: false,
-            street_parking_spot_length: Distance::meters(8.0),
+            parallel_street_parking_spot_length: Distance::meters(8.0),
+            vehicle_width_for_parking_spots: Distance::meters(3.0),
```

Comment:
Just making something up that looks vaguely reasonable. All of this config needs to later turn into region-dependant values